BlueprintUICommonControls/Sources/Internal/KeyboardObserver.swift

```diff
@@ -37,6 +37,13 @@ final class KeyboardObserver {
     init(center : NotificationCenter = .default) {
         
         self.center = center
+        
+        /// We need to listen to both `will` and `keyboardWillChangeFrame` notifications. Why?
```

Comment:
```suggestion
        /// We need to listen to both `will` and `keyboardDidChangeFrame` notifications. Why?
```

I think you meant to make on of these `did`?

Reply:
Doh